cmd/capibmadm/printer/printer.go

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
@dharaneeshvrd I am not sure if we need all this code, see #1075 I have directly used the aws apis and tested all formats including `table`, `json` and `yaml`